tasks/86_working-hours/working-hours.js

```diff
@@ -0,0 +1,44 @@
+export function formatWorkingHours(data) {
+  if (data.length === 0) {
+    return [];
+  }
+  let daysInOrder = ['mon', 'tue', 'wed', 'thu', 'fri', 'sat', 'sun'];
+  daysInOrder = daysInOrder.map((day) => {
+    return data.find((dayObj) => {
+      return dayObj['day'] === day;
+    });
+  });
+  let result = [];
+  for (let i = 0; i < daysInOrder.length; i++) {
+    let schedulePart = [daysInOrder[i]];
+    for (let j = i + 1; j < daysInOrder.length; j++) {
+      if (daysInOrder[i]['from'] === daysInOrder[j]['from']) {
```

Comment:
you need to check also `to` as if it opened to different hour it works different hours